tigerpath/majors_and_certificates/majors/AAS.json

```diff
@@ -20,20 +20,6 @@
     }
   ],
   "req_list": [
-    {
-      "name": "Prerequisites",

```

Comment:
https://aas.princeton.edu/academics/undergraduate-program/concentration-requirements

"The steps to complete the concentration are as follows:
Students complete two core survey courses listed below.  At least one of these must be a Pre-20th century course."